commit a5bdc248592b51ab672ba7b51db0bd32caeaac8a
Author: Pascal Rapicault <prapicau>
Date:   Tue May 10 14:31:02 2011 +0000

    Bug 345209 - [import/export] VersionInCompatibleException spelt wrong

1	1	bundles/org.eclipse.equinox.p2.tests/src/org/eclipse/equinox/p2/tests/importexport/ImportExportTests.java
1	1	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/P2ImportExport.java
0	24	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionInCompatibleException.java
24	0	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionIncompatibleException.java
2	2	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportPage.java
2	2	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/persistence/P2FParser.java
diff --git a/bundles/org.eclipse.equinox.p2.tests/src/org/eclipse/equinox/p2/tests/importexport/ImportExportTests.java b/bundles/org.eclipse.equinox.p2.tests/src/org/eclipse/equinox/p2/tests/importexport/ImportExportTests.java
index dbdebe4..b6ae1a5 100644
--- a/bundles/org.eclipse.equinox.p2.tests/src/org/eclipse/equinox/p2/tests/importexport/ImportExportTests.java
+++ b/bundles/org.eclipse.equinox.p2.tests/src/org/eclipse/equinox/p2/tests/importexport/ImportExportTests.java
@@ -86,7 +86,7 @@ public class ImportExportTests extends AbstractProvisioningTest {
 		try {
 			importexportService.importP2F(input);
 			assertTrue("Didn't complain the given file is not supported by current version.", false);
-		} catch (VersionInCompatibleException e) {
+		} catch (VersionIncompatibleException e) {
 			// expected
 		} finally {
 			input.close();
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/P2ImportExport.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/P2ImportExport.java
index 531192c..1c19bb4 100755
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/P2ImportExport.java
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/P2ImportExport.java
@@ -22,7 +22,7 @@ public interface P2ImportExport {
 	 * @param input
 	 * @return iu listed in the file
 	 * @throws IOException
-	 * @throws VersionInCompatibleException if the given file version is not supported
+	 * @throws VersionIncompatibleException if the given file version is not supported
 	 */
 	List<IUDetail> importP2F(InputStream input) throws IOException;
 
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionInCompatibleException.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionInCompatibleException.java
deleted file mode 100644
index 3759a42..0000000
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionInCompatibleException.java
+++ /dev/null
@@ -1,24 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2011 WindRiver Corporation and others.
- * All rights reserved. This program and the accompanying materials 
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- * 
- * Contributors:
- *     WindRiver Corporation - initial API and implementation
- *******************************************************************************/
-package org.eclipse.equinox.internal.p2.importexport;
-
-public class VersionInCompatibleException extends RuntimeException {
-
-	public VersionInCompatibleException(String message) {
-		super(message);
-	}
-
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = -7584242899366859010L;
-
-}
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionIncompatibleException.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionIncompatibleException.java
new file mode 100644
index 0000000..4a5493b
--- /dev/null
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/VersionIncompatibleException.java
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2011 WindRiver Corporation and others.
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ *     WindRiver Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.equinox.internal.p2.importexport;
+
+public class VersionIncompatibleException extends RuntimeException {
+
+	public VersionIncompatibleException(String message) {
+		super(message);
+	}
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -7584242899366859010L;
+
+}
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportPage.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportPage.java
index 589da37..e00f55f 100644
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportPage.java
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportPage.java
@@ -17,7 +17,7 @@ import java.util.*;
 import java.util.List;
 import org.eclipse.core.runtime.*;
 import org.eclipse.equinox.internal.p2.importexport.IUDetail;
-import org.eclipse.equinox.internal.p2.importexport.VersionInCompatibleException;
+import org.eclipse.equinox.internal.p2.importexport.VersionIncompatibleException;
 import org.eclipse.equinox.internal.p2.importexport.internal.Messages;
 import org.eclipse.equinox.internal.p2.ui.ProvUI;
 import org.eclipse.equinox.internal.p2.ui.ProvUIMessages;
@@ -188,7 +188,7 @@ public class ImportPage extends AbstractImportPage implements ISelectableIUsPage
 				features = importexportService.importP2F(input);
 				viewer.setInput(features.toArray(new IUDetail[features.size()]));
 				input.close();
-			} catch (VersionInCompatibleException e) {
+			} catch (VersionIncompatibleException e) {
 				MessageDialog.openWarning(getShell(), Messages.ImportPage_TITLE, e.getMessage());
 			} catch (FileNotFoundException e) {
 				MessageDialog.openError(getShell(), Messages.ImportPage_TITLE, Messages.ImportPage_FILENOTFOUND);
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/persistence/P2FParser.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/persistence/P2FParser.java
index d1babd0..24c2f74 100644
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/persistence/P2FParser.java
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/persistence/P2FParser.java
@@ -16,7 +16,7 @@ import java.util.ArrayList;
 import java.util.List;
 import javax.xml.parsers.ParserConfigurationException;
 import org.eclipse.equinox.internal.p2.importexport.IUDetail;
-import org.eclipse.equinox.internal.p2.importexport.VersionInCompatibleException;
+import org.eclipse.equinox.internal.p2.importexport.VersionIncompatibleException;
 import org.eclipse.equinox.internal.p2.importexport.internal.Messages;
 import org.eclipse.equinox.internal.p2.persistence.XMLParser;
 import org.eclipse.equinox.p2.metadata.*;
@@ -152,7 +152,7 @@ public class P2FParser extends XMLParser implements P2FConstants {
 			if (P2F_ELEMENT.equals(target)) {
 				Version repositoryVersion = extractPIVersion(target, data);
 				if (!XML_TOLERANCE.isIncluded(repositoryVersion)) {
-					throw new VersionInCompatibleException(NLS.bind(Messages.io_IncompatibleVersion, repositoryVersion, XML_TOLERANCE));
+					throw new VersionIncompatibleException(NLS.bind(Messages.io_IncompatibleVersion, repositoryVersion, XML_TOLERANCE));
 				}
 			}
 		}
